public/app/features/alerting/unified/RuleViewer.tsx

```diff
@@ -36,7 +36,7 @@ const RuleViewer = (): JSX.Element => {
   }, [id]);
 
   // we then fetch the rule from the correct API endpoint(s)
-  const { loading, error, result: rule } = useCombinedRule({ ruleIdentifier: identifier, limitAlerts });
+  const { loading, error, result: rule, uninitialized } = useCombinedRule({ ruleIdentifier: identifier, limitAlerts });
```

Comment:
Nit: Can I suggest `isUninitialized` for naming consistency with "vanilla" RTKQ hooks?